index.html

```diff
@@ -4,47 +4,192 @@
+                        <h4>2015</h4>
+                    </div>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                </div>
+                <ul class="styled-links">
+                        <li><a href="#">html</a></li>
+                        <li><a href="#">css</a></li>
+                        <li><a href="#">JavaScript</a></li>
+                </ul>
+                <ul class="btn">
+                    <li><a href="#">See Project</a></li>
+                </ul>
+            </div>
+            <section>
+              <div class="profile-card">
+                <div class="header-text">
+                  <h2>
+                      About<br>
+                      Myself
+                  </h2>
+                </div>
+                <p>
+                    Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+                </p>
+                <div class="connect-img"><img src="assets/connect.png" alt=""/></div>
+                <div class="social-media-icons">
+                    <ul>
+                        <li><a href="#"><i class="fab fa-twitter"></i></a></li>
+                        <li><a href="#"><i class="fab fa-linkedin"></i></a></li>
+                        <li><a href="#"><i class="fab fa-medium-m"></i></a></li>
+                        <li><a href="#"><i class="fab fa-github"></i></a></li>
+                        <li><a href="#"><i class="fab fa-angellist"></i></a></li>
+                      </ul>
+                </div>
+                <ul class="btn">
+                    <li><a href="#">Get Resume</a></li>
+                </ul>
+                <p>&nbsp;</p>
+                <div class="specials"><img src="assets/Languages.png" alt=""/><a href="#"><i class="fa fa-angle-down icons" aria-hidden="true"></i></a></div>
+                <br />
+                <div class="langs">
+                  <ul>
+                    <li>><a href="#"><img src="assets/js-logo.png" alt="" /> &nbsp; &nbsp; JavaScript</a></li>
+                    <li><a href="#"><img src="assets/html-logo.png" alt="" /> &nbsp; &nbsp;HTML</a></li>
+                    <li><a href="#"><img src="assets/css-logo.png" alt="" /> &nbsp; &nbsp; CSS</a></li>
+                  </ul>
+                </div>
+                <br/>
+                <div class="specials"><img src="assets/Frameworks.png" alt=""><a href="#"><i class='fas fa-greater-than icons'></i></a></div>
+                <div><img src="assets/separator.png" alt=""/></div>
+                <div class="specials"><img src="assets/Skills.png" alt=""/><a href="#"><i class='fas fa-greater-than icons'></i></a></div>
+                <div><img src="assets/separator.png"  alt=""/></div>
+            </div>
+          </section>
+        </section>
```

Comment:
Both work and the about-me section are supposed to be two different sections but you have wrapped both in a single section. Please wrap both of them in two different `section` tags.

Reply:
I have separated the work section from the about-me section as recommended. Thank you so very much for your professional review. I look forward to receiving positive feedback from you so I can prepare for the next step. Thanks in anticipation and much appreciation from me to you